README.org

```diff
@@ -73,7 +74,7 @@ File me [[https://github.com/dennyzhang/cheatsheet.dennyzhang.com/issues][Issues
 | Delete all resources filtered by labels | =kubectl delete pods,services -l name=myLabel=           |
 | Delete resources under a namespace      | =kubectl -n my-ns delete po,svc --all=                   |
 | Delete persist volumes by labels        | =kubectl delete pvc -l app=wordpress=                    |
-| Delete statefulset only (not pods)      | =kubectl delete sts/<stateful_set_name> --cascade=false= |
+| Delete state fulset only (not pods)     | =kubectl delete sts/<stateful_set_name> --cascade=false= |
```

Comment:
@tehtbl , sorry I'm late for the review. 

Looks like we have an unexpected whitespace for the word of statefulset. Could you help to update the PR for this?

Reply:
No worries. I'm adding the changes. Thanks for review!

Reply:
Thanks for the help! The changes look good to me. Let's merge it.